lib/math/math.go

```diff
@@ -0,0 +1,100 @@
+package math
+
+import (
+	"math"
+
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+)
+
+const tau = math.Pi * 2
+const oneRad = tau / 360
+
+// Module math is a Starlark module of math-related functions.
+var Module = &starlarkstruct.Module{
+	Name: "math",
+	Members: starlark.StringDict{
+		"ceil":  starlark.NewBuiltin("ceil", ceil),
+		"fabs":  starlark.NewBuiltin("fabs", fabs),
+		"floor": starlark.NewBuiltin("floor", floor),
+		"round": starlark.NewBuiltin("round", round),
+
+		"exp":  starlark.NewBuiltin("exp", exp),
+		"sqrt": starlark.NewBuiltin("sqrt", sqrt),
+
+		"acos":  starlark.NewBuiltin("acos", acos),
+		"asin":  starlark.NewBuiltin("asin", asin),
+		"atan":  starlark.NewBuiltin("atan", atan),
+		"atan2": starlark.NewBuiltin("atan2", atan2),
+		"cos":   starlark.NewBuiltin("cos", cos),
+		"hypot": starlark.NewBuiltin("hypot", hypot),
+		"sin":   starlark.NewBuiltin("sin", sin),
+		"tan":   starlark.NewBuiltin("tan", tan),
+
+		"degrees": starlark.NewBuiltin("degrees", degrees),
```

Comment:
These don't seem necessary. It's easy to say `theta*60/(2*pi)` on the rare occasion that it is needed.

Reply:
Please keep them as code using it is much easier to understand.